Controllers/ItemsController.cs

```diff
@@ -22,7 +22,7 @@ public ItemsController(ApplicationDbContext context)
         }
 
         [LogAspect]
-        private int AggregateSuming(List<ItemDTO> items)
+        private int AggregateSuming(List<ItemDTO> items, int someting=1)
```

Comment:
I guess spacing is da problem :) "int something=1" should be "int something = 1"

Reply:
this argument was only for testing purposes, it will be deleted :D